Testcases/UITests/uiPopupTest.robot

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Documentation          This testcase validates the popup messages of VX
+...                     Environment: ${host_ip}
+...                     Group Name: ${group_name}
+Resource    ${EXECDIR}/Resources/apiresources.robot
+Resource    ${EXECDIR}/Resources/uiresources.robot
+Resource    ${EXECDIR}/Resources/connection.robot
+Resource    ${EXECDIR}/Resources/UIResources/uiPopupResources.robot
+Resource    ${EXECDIR}/Inputs/testInputs.robot
+
+
+*** Test Cases ***
+VXPopupMessages
+    [Setup]    popupResources.popupTestSetup
+    popupResources.suppressAlarmPopup
+
+CXPopupMessages
+    log to console  override
```

Comment:
update log to console   CXPopupMessages yet to be implemented  and comment it or remove it